presto-spark-base/src/main/java/com/facebook/presto/spark/PrestoSparkInjectorFactory.java

```diff
@@ -189,6 +184,16 @@ public Injector create()
                     injector.getInstance(SessionPropertyDefaults.class).loadConfigurationManager();
                 }
             }
+
+            if (sparkProcessType.equals(DRIVER) ||
+                    configProperties.getOrDefault(INLINE_SQL_FUNCTIONS, "true").equals("false")) {
```

Comment:
You can get an instance of a configuration class by using `injector.getInstance(...)`